t/atnodes.t

```diff
@@ -56,6 +56,8 @@ OPTIONS:
     -v            Be verbose.
     -w            Prompt for password (used for login and sudo,
                   could be privided by SSH_BATCH_PASSWORD).
+    -W            Prompt for password (like -w, but used for sudo only,
+                  when -W is on, -w will be ignored).
```

Comment:
Hmm, maybe it's better to complain about the appearance of both `-w` and `-W`? They're conflicting options per se :)